frontend/src/v5/ui/components/shared/breadcrumbs/breadcrumbs.component.tsx

```diff
@@ -32,14 +37,17 @@ export const Breadcrumbs = (): JSX.Element => {
 
 				if (isLastItem) {
 					return (
-						<InteractiveBreadcrumb onClick={handleClick} endIcon={<ExpandMoreIcon />}>
-							{title}
-						</InteractiveBreadcrumb>
+						<div>
+							<InteractiveBreadcrumb onClick={handleClick} endIcon={<ExpandMoreIcon />}>
+								{title}
+							</InteractiveBreadcrumb>
+							<NavigationMenu anchorEl={anchorEl} handleClose={handleClose} />
```

Comment:
I think that the options for navigation menu should be passed here instead of being managed inside navigationmenu